hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -0,0 +1,553 @@
+                }
+                });
+
+        }); // $(document).ready(function())
+
+
+        function remove_collection_item(res_id)
+        {
+            // get row obj is being removed from collection
+            var removed_row_from_colletion = $('#collection-table').DataTable().row('#' + res_id);
+
+            if(removed_row_from_colletion.data()[5].toLowerCase().indexOf("none") >= 0) {
+                $('#save-collection-btn-cancel').off("click");
+                $('#save-collection-btn-ok').off("click").on("click", function(){
+                    removed_row_from_colletion.remove().draw(false);
+                });
+
+                waning_message = "You have NO PERMISSION over the resource you are about to remove from this collection. Once confirmed, you CAN NOT add it back.";
+
+                $('#save-collection-warning-body').text(waning_message);
+                $('#save-collection-warning').modal('show');
+            }
+            else
+            {
+                // copy row data and add to collection-candidate table
+                $('#collection-table-candidate').DataTable().row.add(removed_row_from_colletion.data()).draw(false);
+                // remove row obj
+                removed_row_from_colletion.remove().draw(false);
+            }
+            $('#save-new').show();
+        }
+
+
+        function add_collection_item()
+        {
+            $.each($(".row-selector:checked"),function( index, chkbox ) {
+                    var res_id = chkbox.id;
+                    var added_row = $('#collection-table-candidate').DataTable().row('#' + res_id);
+                    $('#collection-table').DataTable().row.add(added_row.data()).draw(false);
+                    added_row.remove().draw(false);
+            });
+            $('#save-new').show();
+        }
+
+        // revised metadata_update_ajax_submit
+        function collection_update_ajax_submit_new(form_id)
+        {
+            $alert_success = '<div class="alert alert-success" id="success-alert"> \
+                <button type="button" class="close" data-dismiss="alert">x</button> \
+                <strong>Success! </strong> \
+                Collection updated.\
+            </div>'
+            $alert_error = '<div class="alert alert-danger" id="error-alert"> \
+                <button type="button" class="close" data-dismiss="alert">x</button> \
+                <strong>Error! </strong> \
+                Collection failed to update.\
+            </div>'
+            $form=$('#' + form_id);
+            var resource_id_list = [];
+            $('#collection-table').DataTable().rows().every(function()
+            {resource_id_list.push(this.id);});
+            $.ajax({
+                type: "POST",
+                url: $form.attr('action'),
+                dataType: "json",
+                data: {"resource_id_list": resource_id_list},
+                success: function(result)
+                {
```

Comment:
@zhiyuli move this { to its previous line after function(result) to have consistent coding format